src/client/sandbox/shadow-ui.js

```diff
@@ -393,8 +386,19 @@ export default class ShadowUI extends SandboxBase {
         return names.join(' ');
     }
 
-    getUIStylesheet () {
-        return nativeMethods.querySelector.call(this.document, 'link.' + SHADOW_UI_CLASS_NAME.uiStylesheet);
+    getUIStylesheetsCopyFragment () {
+        var stylesheets         = Array.prototype.slice.call(
+            nativeMethods.querySelectorAll.call(this.document, 'link.' + SHADOW_UI_CLASS_NAME.uiStylesheet));
+        var fragment = document.createDocumentFragment();
+
+        if (stylesheets.length)
+            stylesheets.forEach(stylesheet => fragment.appendChild(stylesheet.cloneNode(true)));
```

Comment:
`Array.prototype.slice.call` is used in many places.

![1](https://cloud.githubusercontent.com/assets/4133518/11030461/3f33d9b6-86df-11e5-96f3-ca6b5d88654b.png)

I think we need to create `cloneItems(iterableObject)` util and rewrite all places.
/cc @inikulin